assets/scripts/store/reducers/settings.js

```diff
@@ -0,0 +1,28 @@
+import { SET_USER_SETTINGS } from '../actions'
+// Note: turning this constant on runs too much side-effect code too early.
```

Comment:
Ah I see. Is it that the imports in ```../../streets/creation``` run side effects? I'm not suggesting fixing the side effects right now, but maybe at least taking inventory of them and what needs to be done? Updating the comment I think would be good enough for now